### PR TITLE
Clean up collector keys in values.yaml

### PR DESCRIPTION
**Pull request by pmalek-sumo**
###### Description

Fill in your description here.

###### Testing performed

- [ ] ci/build.sh
- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

File: tests/upgrade_v2_script/static/sumologic_collector_fields.input.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+          memory: 64Mi
+          cpu: 200m
+      nodeSelector: {}
+      ## Add custom labels only to setup job pod
+      podLabels: {}
+      ## Add custom annotations only to setup job pod
+      podAnnotations: {}
+    ## Configuration of additional collector fields
+    ## https://help.sumologic.com/Manage/Fields#http-source-fields
+    fields: {}
```
Comment (sumo-drosiek) [inline tests/upgrade_v2_script/static/sumologic_collector_fields.input.yaml:36]: add newline before EOF

**Review by sumo-drosiek — COMMENTED** [review]
Can we stick to one list format?
you changes from `-` to `*` and in another file you introduces `-`

Pretty confusing

**Comment by pmalek-sumo** [comment]
> Can we stick to one list format?
> you changes from `-` to `*` and in another file you introduces `-`
> 
> Pretty confusing

`mdl` warns about inconsistent usage of list bullets in single file not across files. Not sure if we can enforce it.

**Comment by sumo-drosiek** [comment]
would be nice to be consistent across the PR

**Review by sumo-drosiek — APPROVED** [review]

**Pull request merged**